cannot build (ApplicationInsights.config not found)

i cloned this project, opened up the .sln with VS 2015, pressed F5 and it does not build:

```
1>------ Erstellen gestartet: Projekt: PieEatingNinjas.PivotWithoutSwipe, Konfiguration: Debug x86 ------
1>C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(4106,5): error MSB3030: Datei "C:\Users\XXXXX\repositories\PivotWithoutSwipe\PieEatingNinjas.PivotWithoutSwipe\PieEatingNinjas.PivotWithoutSwipe\ApplicationInsights.config" konnte nicht kopiert werden, da die Datei nicht gefunden wurde.
========== Erstellen: 0 erfolgreich, 1 fehlerhaft, 0 aktuell, 0 übersprungen ==========
```

The error message says "[] could not be copied, because the file was not found".
Am i doing something wrong?

--- a/PieEatingNinjas.PivotWithoutSwipe/PieEatingNinjas.PivotWithoutSwipe/App.xaml.cs
+++ b/PieEatingNinjas.PivotWithoutSwipe/PieEatingNinjas.PivotWithoutSwipe/App.xaml.cs
@@ -28,9 +28,6 @@ namespace PieEatingNinjas.PivotWithoutSwipe
         /// </summary>
         public App()
         {
-            Microsoft.ApplicationInsights.WindowsAppInitializer.InitializeAsync(
-                Microsoft.ApplicationInsights.WindowsCollectors.Metadata |
-                Microsoft.ApplicationInsights.WindowsCollectors.Session);
             this.InitializeComponent();
             this.Suspending += OnSuspending;
         }

--- a/PieEatingNinjas.PivotWithoutSwipe/PieEatingNinjas.PivotWithoutSwipe/PieEatingNinjas.PivotWithoutSwipe.csproj
+++ b/PieEatingNinjas.PivotWithoutSwipe/PieEatingNinjas.PivotWithoutSwipe/PieEatingNinjas.PivotWithoutSwipe.csproj
@@ -88,13 +88,6 @@
     <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
   </PropertyGroup>
   <ItemGroup>
-    <!-- A reference to the entire .Net Framework and Windows SDK are automatically included -->
-    <Content Include="ApplicationInsights.config">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <None Include="project.json" />
-  </ItemGroup>
-  <ItemGroup>
     <Compile Include="App.xaml.cs">
       <DependentUpon>App.xaml</DependentUpon>
     </Compile>

--- a/PieEatingNinjas.PivotWithoutSwipe/PieEatingNinjas.PivotWithoutSwipe/project.json
+++ b/PieEatingNinjas.PivotWithoutSwipe/PieEatingNinjas.PivotWithoutSwipe/project.json
@@ -1,8 +1,5 @@
 ﻿{
   "dependencies": {
-    "Microsoft.ApplicationInsights": "1.0.0",
-    "Microsoft.ApplicationInsights.PersistenceChannel": "1.0.0",
-    "Microsoft.ApplicationInsights.WindowsApps": "1.0.0",
     "Microsoft.NETCore.UniversalWindowsPlatform": "5.0.0"
   },
   "frameworks": {

